xml/System.Runtime.Serialization.Formatters.Binary/BinaryFormatter.xml

```diff
@@ -285,7 +285,7 @@
 ## Remarks  
   
 > [!IMPORTANT]
->  Calling this method with untrusted data is a security risk. Call this method only with trusted data. For more information, see [Untrusted Data Security Risks](http://go.microsoft.com/fwlink/?LinkId=330378).  
+>  Calling this method with untrusted data is a security risk. Call this method only with trusted data. For more information, see [Untrusted Data Security Risks](https://www.owasp.org/index.php/Data_Validation).  
```

Comment:
Since you changed the link to a new target, you should also change the link text to match the title of that page. 

Untrusted Data Security Risks -> Data Validation